guides/v2.3/performance-best-practices/advanced-js-bundling.md

```diff
@@ -479,11 +479,11 @@ Open `requirejs-config.js` in the `pub/static/frontend/Magento/luma/en_US` direc
 
 ```javascript
 require.config({
-    bundles: { "bundles/default": ["mage/template","mage/apply/scripts","mage/apply/main","mage/mage","mage/translate","mage/loader"] }
-});
-require.config({
-    bundles: { "bundles/cart": ["Magento_Ui/js/lib/validation/utils","Magento_Ui/js/lib/validation/rules","Magento_Ui/js/lib/validation/validation"] }
-});
+    bundles: {
+        "bundles/default": ["mage/template", "mage/apply/scripts", "mage/apply/main", "mage/mage", "mage/translate", "mage/loader"],
+        "bundles/cart": ["Magento_Ui/js/lib/validation/utils", "Magento_Ui/js/lib/validation/rules", "Magento_Ui/js/lib/validation/validation"]
+    }
+}
```

Comment:
Bundles are now written in a single require.config instruction. This output might contain more spacing (one module per line) depending on whether you're using node.js, rhino or something else.